target/linux/sunxi/image/Makefile

```diff
@@ -11,14 +11,13 @@ include $(INCLUDE_DIR)/image.mk
 FAT32_BLOCK_SIZE=1024
 FAT32_BLOCKS=$(shell echo $$(($(CONFIG_SUNXI_SD_BOOT_PARTSIZE)*1024*1024/$(FAT32_BLOCK_SIZE))))
 
-KERNEL_LOADADDR:=0x40008000
+KERNEL_LOADADDR:=0x40080000
```

Comment:
I noticed that kernel_addr_r is different for non 64-bit platforms. So KERNEL_LOADADDR have to be defined per platform type.

https://github.com/u-boot/u-boot/blob/master/include/configs/sunxi-common.h#L319